### PR TITLE
add: `asdf-vm`

### PR DESCRIPTION
**Pull request by niontrix**
Add asdf version manager package.
For reference see https://asdf-vm.com

## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packages/asdf-vm/asdf-vm.pacscript [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  cp -r bin "${dst}"
+  cp -r lib "${dst}"
+  cp asdf.elv "${dst}"
+  cp asdf.fish "${dst}"
+  cp asdf.nu "${dst}"
+  cp asdf.sh "${dst}"
+  cp defaults "${dst}"
+  cp help.txt "${dst}"
+  cp version.txt "${dst}"
```
[the remark this answers is not in the document]
Comment (niontrix) [reply packages/asdf-vm/asdf-vm.pacscript]: Done. Thanks for the feedback!

**Review by vigress8 — DISMISSED** [review]

File: packages/asdf-vm/asdf-vm.pacscript [hunk 2]
```diff
@@ -0,0 +1,65 @@
+pkgname=asdf-vm
+pkgver=0.14.1
+pkgrel=1
```
Comment (Elsie19) [inline packages/asdf-vm/asdf-vm.pacscript:3]:
```suggestion
pkgname="asdf-vm"
pkgver="0.14.1"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Can you put the repology key in here?

**Comment by niontrix** [comment]
@Elsie19  unfortunately I get the error messages `Cannot find repology` when trying to `pacup -r asdf-vm.pacscript`. Do you have any hints what the problem might be?

**Comment by vigress8** [comment]
That's not what `-r` does - it prints the Repology API response using the existing metadata from the `repology` variable.
Add `repology=("project: ${gives}") to the pacscript

**Comment by niontrix** [comment]
Sorry, I have trouble understanding the process.
At the moment there is no `gives` variable in my pacscript. Do I need to add it with `gives="asdf"`?
Also, do you mean I should not run `pacup -r asdf-vm.pacscript`? If so, why not? I thought this would show me that it is able to find a package that pacup could then use to update the version and other information for a next release, does it not?

**Comment by niontrix** [comment]
But how do I know that it works like it should? That's why I wanted to see the output of `pacup --show-repology`.

File: packages/asdf-vm/asdf-vm.pacscript [hunk 3]
```diff
@@ -0,0 +1,65 @@
+pkgname=asdf-vm
+pkgver=0.14.1
+pkgdesc='Extendable version manager with support for Ruby, Node.js, Elixir, Erlang & more'
+repology=("project: ${pkgname})
```
Comment (vigress8) [inline packages/asdf-vm/asdf-vm.pacscript:4]:
```suggestion
repology=("project: ${pkgname}")
```

**Review by vigress8 — APPROVED** [review]

**Comment by niontrix** [comment]
@Elsie19 is there still something missing or something I need to do?

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**